libzhlgen/test/functions/win32/1.6.9/OptionsScreen.zhl

```diff
@@ -4,3 +4,8 @@ __thiscall void OptionsScreen::OnInit(OptionsScreen *this);
 __thiscall void OptionsScreen::OnLoop(OptionsScreen *this);
 "57660f76c08d7c240883e4f0ff77fc5589e557565381ec9c0400008b07898d70fbffff0f1181f8090000c681cc02000000":
 __thiscall void OptionsScreen::Open(OptionsScreen *this, bool mainMenu);
+"57????????83e4f0ff77fc5589e557565389ce8d4d9081ecac000000":
```

Comment:
This should be in its own file `LanguageChooser.zhl` to be consistent with how we were structuring all the other files.

`LanguageChooser::OnRender`
Linux:
64: `4157bef001000041564155`
32: `5557565331db83ec6c`